library/src/main/java/dev/chrisbanes/insetter/Insetter.java

```diff
@@ -80,7 +227,7 @@ public WindowInsetsCompat onApplyWindowInsets(View v, WindowInsetsCompat insets)
    * A wrapper around {@link ViewCompat#requestApplyInsets(View)} which ensures the request will
    * happen, regardless of whether the view is attached or not.
    */
-  public static void requestApplyInsetsWhenAttached(@NonNull final View view) {
+  private static void requestApplyInsetsWhenAttached(@NonNull final View view) {
```

Comment:
It doesn't really seem necessary to expose this. Can revert if needed.

Reply:
👍